main.js

```diff
@@ -24,4 +24,195 @@ window.onload = () => {
     theHead.classList.toggle('make_blur');
     myShow.classList.toggle('show');
   });
-};
\ No newline at end of file
+
+    
+const store = [{
+    detail1:'key1', 
+    name: 'Tonic',
+    img: 'images/Project_1.svg',
+    img_desk: 'images/card_1_image_desk.svg',
+    name_desk: 'Tonic',
+    companyName: 'CANOPY',
+    companyNameDesk: 'Canopy',
+    job: 'Back End Dev',
+    jobDesk: 'Back End Dev',
+    year: 2015,
+    yearDesk: 2015,
+    description :'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    tags: ['html', 'css', 'javaScript'],
+
+  },
```

Comment:
- [ ] Great work making this an array of objects. One thing is left to do, add the key-value pairs for the live-link version and the source link. Currently, you have these two listed in the HTML modal structure.